### ISSUE TITLE
Easy way to store predictions?

### ISSUE THREAD

**Issue by EmmaRocheteau** [MIC-DKFZ/trixi#25]
Hi! Thanks for making such a great tool! I've had the pleasure of exploring it over the weekend. My question about whether it's easy to store the predictions made by the model? Or should I write my own code for this and make a new folder etc. Just want to check whether you've already made a framework for this

**Comment by jenspetersen** [comment]
Hi Emma,

glad you like trixi :) I'm assuming you're using an ExperimentLogger in some way (PytorchExperimentLogger is a subclass, and PytorchExperiment automatically uses it)? The ExperimentLogger has a method `save_numpy_data` that you could use, it writes to the "save" folder in your experiment, see [here](https://trixi.readthedocs.io/en/develop/_api/trixi.logger.html#trixi.logger.experiment.experimentlogger.ExperimentLogger.save_numpy_data). You can also write to a subfolder in "save" like this

    myexplogger.save_numpy_data(model_prediction.detach().cpu().numpy(), "predictions/epochXYZ.npy")

when you're on Linux (otherwise use "\\\\" instead of "/"). Hope that helps?

Jens

**Comment by EmmaRocheteau** [comment]
Yup! I'm following the PyTorchExperiment format, so I've written my own setup, train, validate and test functions. That's fantastic! You've saved me a bunch of time thank you.
